### ISSUE TITLE
For scientific writing support of many kinds of LaTeX math blocks is needed

### ISSUE THREAD

**Issue by N0rbert** [Zettlr/Zettlr#686]
## Description

Many scientific texts contain equations. Currently texts are sometimes written in RMarkdown.

Some texts are books and articles written in [bookdown](https://bookdown.org/yihui/bookdown/markdown-extensions-by-bookdown.html#equations), some of texts are presentations written with [xaringan](https://bookdown.org/yihui/rmarkdown/xaringan.html).

## Example Workflow

Have the following RMarkdown document with contents below:

```
Block LaTeX equation without auto-numbering:

$$ g(x) = \cos(x)$$

---

Environment `\begin{equation}...\end{equation}`
with auto-numbering:

\begin{equation}
h(z) = \sinh(z)
\end{equation}

---

Environment `\begin{equation}...(\#eq:label)\end{equation}`
with auto-numbering and label (for bookdown):

\begin{equation}
  f\left(k\right) = \binom{n}{k} p^k\left(1-p\right)^{n-k}
  (\#eq:binom)
\end{equation}

---

Environment `\begin{equation}...\label{label}\end{equation}`
with auto-numbering label for xaringan

\begin{equation}
  g\left(k\right) = \binom{n}{k} p^k\left(1-p\right)^{n-k}
\label{binom2}
\end{equation}
```

## Additional Information

This functionality is currently supported in VNote and ReText.
In this feature request I ask only for rendering of equations. 
Auto-numbering is not needed for me.
I'll do it in RStudio after knitting of the document.

**Comment by nathanlesage** [comment]
Writing equations is already supported the way Pandoc does it, that is: Single dollar signs for inline math equations and double-dollar signs for display math equations (the latter will apparently receive even better support after one of the current PRs is being merged, but I'll have to see to that).

Simply make sure you have "render equations" set to true in the Preferences -> Display settings.

Equations outside of dollar-signs, however, is not supported and I am not planning on supporting special documents that are not exportable using Pandoc. However, given the fact that you can use Pandoc in conjunction with RStudio, this form of knitting is supported, and you can activate RMarkdown-support in the preferences under advanced!

Concerning auto-numbering: This is apparently, if I'm not mistaken, supported by a Pandoc-plugin, and I'll increase support for Pandoc plugins in future versions.

**Comment by N0rbert** [comment]
*Render Formulae* is enabled, but double dollar is not rendered:

![zettlr-eqs](https://user-images.githubusercontent.com/309456/79787823-45ce6d00-8350-11ea-94f4-36fb0042629b.png)


The three latter equations are supported by Pandoc, export to HTML  and PDF from Zettlr show them correctly. So I would really suggest to support them in preview mode. It would make Zettlr better.

**Comment by N0rbert** [comment]
Above is still actual for 2.0.3.

**Comment by nathanlesage** [comment]
CC @tobiasdiez feel free to open if you see something to fix here.

**Comment by tobiasdiez** [comment]
To be honest, I don't really understand what's the issue. Is it that the double dollar symbols only work if they are on separate lines before and after the equation content?

**Comment by N0rbert** [comment]
Bus is not fixed in the latest Zettlr 3.0.2

**Comment by nathanlesage** [comment]
The issue with the double dollars might just be that you cannot pad the equation. There cannot be a space between the dollars and the equation. If that's the issue since more than a year, it is explicitly mentioned in the documentation.

**Comment by N0rbert** [comment]
Bug is not fixed in the latest Zettlr 3.0.3.
